### PR TITLE
Some documentation: api, manual

### PR DESCRIPTION
**Pull request by rjbou**

### CODE REVIEW

File: doc/pages/Tricks.md [hunk 1]
```diff
@@ -78,6 +80,22 @@ $ opam pin <HTTPS clone URL>
 
 Furthermore, `opam update` will then pick up any modification you made to the gist.
 
+##### Strict dependencies
```
Comment (AltGr) [inline doc/pages/Tricks.md]: I am not sure this is the appropriate spot for this, it should rather be in the FAQ

Comment (rjbou) [reply doc/pages/Tricks.md]: It is in the idea of _sharing setup_, that's why I put it here. We can move the entire section in the FAQ.

File: doc/pages/FAQ.md [hunk 2]
```diff
@@ -172,9 +172,19 @@ opam switch import <file.export> --switch <new switch>
...
-the precise set of installed dependencies for a local package, and the
-associated `opam install DIR --locked` command that can restore them.
+### 🐫 How to share my working switch setup for a specific package ?
+
+When working on a project, it is sometimes needed to share a set of
+dependencies that you know (locally) the project is working with. You can share
+this set by generating a _locked_ opam file. Ths is easil done by the [`lock`
+command](man/opam-lock.html): it creates an opam file with a `depends:` field
+populated with all dependencies, at their exact version in the current
+(working) switch. You just have to share this _locked_ opam file.
```
Comment (AltGr) [inline doc/pages/FAQ.md:182]:
```suggestion
(working) switch. You can then share this `opam.locked` file, or check it
in your version-control system.
```

File: src/core/opamDirTrack.ml [hunk 3]
```diff
@@ -31,12 +31,17 @@ type change =
 
 type t = change SM.t
 
-let string_of_change = function
-  | Added _ -> "addition"
+let string_of_change ?(full=false) =
```
Comment (AltGr) [inline src/core/opamDirTrack.ml:34]: Did you mean to check this part in ?

Comment (rjbou) [reply src/core/opamDirTrack.ml:34]: just a remaining commit...

**Pull request merged**